### PR TITLE
feat: add payment request and payment method

### PR DESCRIPTION
**Pull request by ligertsicat**
Changes:
Add payment requests and payment method docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1601,8 +1615,242 @@ p.createFeeRule(data: {
...
+    currency: PaymentRequestCurrencies;
+    amount: number;
+    reference_id?: string;
+    customer_id?: string;
+    country: PaymentRequestCountries;
+    description?: string;
+    payment_method: object;
+    channel_properties?: PaymentRequestChannelProperties;
+    metadata?: object;
+    idempotencty_key?: string;
```
Comment (cassiewang) [inline README.md:1665]: typo `idempotency_key`

File: README.md [hunk 2]
```diff
@@ -1601,8 +1615,242 @@ p.createFeeRule(data: {
...
+```ts
+r.confirmPaymentRequest(data: {
+    id: string;
+    auth_code: string;
+    idempotencty_key?: string;
+    for_user_id?: string;
+})
+```
+
+#### resend payment request
```
Comment (cassiewang) [inline README.md:1707]: can this say "resend auth for payment request`

File: README.md [hunk 3]
```diff
@@ -1601,8 +1615,242 @@ p.createFeeRule(data: {
...
+    after_id?: string;
+    before_id?: string;
+    created?: string;
+    updated?: string;
+    for_user_id?: string;
+})
+```
+
+
+Refer to [Xendit API Reference](https://developers.xendit.co/api-reference/#refunds) for more info about methods' parameters
```
Comment (cassiewang) [inline README.md:1851]: Is this supposed to link to https://developers.xendit.co/api-reference/#payment-methods ?

File: README.md (hunk 1 above)
Comment (ligertsicat) [reply README.md]: fixed. also update other instances of this

File: README.md (hunk 2 above)
Comment (ligertsicat) [reply README.md]: yeah, updated the function name

File: README.md (hunk 3 above)
Comment (ligertsicat) [reply README.md]: yup my bad. changed to payment methods and added a link for payment request too

**Review by xen-HendryZheng — CHANGES_REQUESTED** [review]
Please bump up package.json @ligertsicat

**Comment by ligertsicat** [comment]
Hello @xen-HendryZheng 
I've updated the package json

**Review by xen-HendryZheng — APPROVED** [review]

**Pull request merged**